test/project_types/script/forms/ask_org_test.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require "project_types/script/test_helper"
+
+describe Script::Forms::AskOrg do
+  include TestHelpers::FakeFS
+  describe ".ask" do
+    let(:context) { TestHelpers::FakeContext.new(root: Dir.mktmpdir) }
```

Comment:
Regarding `Dir.mktmpdir`: maybe this is why `FakeFS` is used.  I don't think we want to actually create any new files.  Can we just try to use `Mock` here?  We explicitly `.expects` or `.stubs` any methods on `context` anyway.  FYI, CLI team is planning on getting rid of this `Context` object eventually.

Reply:
Discussed in meet, we'll avoid doing this here to limit the scope creep of this PR.